youtube_dl/extractor/canvas.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+
+
+class CanvasIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)?(?:www\.)?canvas\.be/video/(?P<id>.+)'
+    _TEST = {
+        'url': 'http://www.canvas.be/video/slijk/2015/isaac-kimeli',
```

Comment:
@dstftw I tried multiple other videos with a proxy server and none would show up. Is it possible that all videos on this website have georestriction? And in this case, should I remove the test? I implemented your other remarks and everything works fine.